openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -253,7 +253,6 @@ <h1 translate>Agenda</h1>
           uib-tooltip="{{ 'Projector' | translate }} {{ item.isProjected(item.tree)[0] || '' }}"
           tooltip-enable="item.isProjected(item.tree).length">
           <button class="btn btn-default btn-sm"
-              title="{{ 'Project item' | translate }}"
```

Comment:
Why did you delete the whole line?

Reply:
Because we have no tooltips for project button in other list views. Tooltips are used only in the buttons on top of the page - not in the list view tables (which would overlaps the row entries).